Resources/views/backend/config/view/form/document_paypal_unified.js

```diff
@@ -16,10 +16,21 @@ Ext.define('Shopware.apps.Config.view.form.DocumentPaypalUnified', {
      * @return { Array }
      */
     getFormItems: function() {
-        var me = this,
-            data = me.callParent(arguments);
+        var formItems = this.callParent(arguments);
 
-        data[1].items.push({
+        var elementFieldSetIndex = -1;
+        Ext.Array.each(formItems, function (item, index) {
```

Comment:
Sorry for just noticing this: Please use `forEach` as provided by `Array.prototype` instead of ExtJS's polyfill `Ext.Array.each`. It is available in all browsers.

```suggestion
        formItems.forEach(function (item, index) {
```